PublicSDKAPITester/APITester.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  APITester.swift
+//  APITester
+//
+//  Created by Joshua Liebowitz on 6/24/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+// This is fine 🔥
```

Comment:
I think it'd be a good idea to document the purpose of this file, otherwise the next dev might be tempted to just kill it, since this isn't really a Swift app. 
how about replacing this comment with
```
// This file is empty, but is needed so that Xcode can correctly
// link Swift dependencies for the framework.
```
?

Reply:
True true

Reply:
✅